althea_kernel_interface/src/exit_server_tunnel.rs

```diff
@@ -89,6 +129,88 @@ impl dyn KernelInterface {
             ],
         )?;
 
+        // Set up link local mesh ip in wg_exit as fe80 + rest of mesh ip of exit
+        let local_link = to_wg_local(&exit_mesh);
+
+        let _output = self.run_command(
+            "ip",
+            &[
+                "address",
+                "add",
+                &format!("{}/64", local_link),
+                "dev",
+                "wg_exit",
+            ],
+        )?;
+
+        // Add iptable routes between wg_exit and eth0
+        let output = self
+            .run_command(
+                "ip6tables",
+                &[
+                    "-C",
+                    "FORWARD",
+                    "-i",
+                    "wg_exit",
+                    "-o",
+                    &external_nic,
+                    "-j",
+                    "ACCEPT",
+                ],
+            )
+            .unwrap();
+        if !output.stderr.is_empty() {
+            let res = self.run_command(
+                "ip6tables",
+                &[
+                    "-A",
+                    "FORWARD",
+                    "-i",
+                    "wg_exit",
+                    "-o",
+                    &external_nic,
+                    "-j",
+                    "ACCEPT",
+                ],
+            );
+            if res.is_err() {
+                error!("IPV6 ERROR: uanble to set ip6table rules");
+            }
+        }
+        let output = self
+            .run_command(
+                "ip6tables",
+                &[
```

Comment:
this looks like it's a re-implemetnation of some of the logic provided for iptables in althea-kernel-interface. Why not duplicate that function s add_ip6tables_rule and avoid these big if-else blocks

Reply:
fixed